src/plugins/visualizations/server/saved_objects/visualization_migrations.ts

```diff
@@ -539,6 +539,39 @@ const migrateTableSplits: SavedObjectMigrationFn = doc => {
   }
 };
 
+const removeMatchAllQueryIfLanguageNotDefined: SavedObjectMigrationFn = doc => {
```

Comment:
You've already defined the same function above. Can we leave only one function and use everywhere?

Reply:
@VladLasitsa  Unfortunately, we cannot do this because we currently do not have a common place for migration scripts associated with various types of saved objects.